client/windows/CCreatureWindow.cpp

```diff
@@ -521,8 +532,16 @@ void CStackWindow::CWindowSection::createCommanderAbilities()
 	}
 }
 
-void CStackWindow::setSelection(si32 newSkill, CClickableObject * newIcon)
+void CStackWindow::setSelection(si32 newSkill, CCommanderSkillIcon * newIcon)
 {
+	auto getSkillDescription = [this](int skillIndex, bool selected) -> std::string
+	{

```

Comment:
This code is duplicated, but could be easily extracted - doesn't event depend on this class.